src/core/mount.c

```diff
@@ -1073,11 +1074,37 @@ static void mount_enter_unmounting(Mount *m) {
 
         assert(m);
 
+        r = path_is_mount_point(m->where, NULL, 0);
```

Comment:
This is still racy. The mount point may be unmounted after `path_is_mount_point()` succeeds and before invoking `umount` command.

Reply:
see comment in commit msg. Yes, the race exists, this PR only addresses it if PID 1 itself is the culprit: we won't race against our own mount attempts anymore. But yes, in the long run we want https://github.com/util-linux/util-linux/issues/2132